libs/nestjs-admin/src/utils/widget.ts

```diff
@@ -139,7 +139,14 @@ export function getWidgetTemplate(column: ColumnMetadata) {
       >
     | 'simple_array'
 
-  const type: ImplementedTypes = column.type as any
+  const type: ImplementedTypes = column.type as ImplementedTypes
+  const name: string = column.propertyName
+
+  switch (name) {
+    case 'password':
```

Comment:
We're basing the type of the field on the name of the column in the db?

Reply:
Yep - right now we don't have another way to distinguish password